index.html

```diff
@@ -6,3 +6,4 @@ <h1> WaziApps examples </h1>
 <li> <a href=www/complex> Retrieve a several sensor values </a> </li>
 <li> <a href=www/graph> Create a graph </a> </li>
 <li> <a href=www/map> Create a map </a> </li>
+<li> <a href=www/complete> Starting boilerplate </a> </li>
```

Comment:
Instead of "Starting boilerplate", why not "Complete webapp example"? Also in the first line, there are now five examples.